x/tx/go.mod

```diff
@@ -1,4 +1,4 @@
-module cosmossdk.io/tx
+module cosmossdk.io/x/tx
```

Comment:
> Should not we keep the full cosmos/cosmos-sdk path for consistency with other modules under x/?

What do you mean by this?

Reply:
I mean that the NFT module name is https://github.com/cosmos/cosmos-sdk/blob/main/x/nft/go.mod#L1.
When we create go.mod for staking et al. it would be tremendously breaking if we rename the package.
So unless we do that, it would be more consistent to have tx called `https://github.com/cosmos/cosmos-sdk/x/tx`.
This is only a nit for consistency’ sake.

Reply:
Ahhh I totally agree here!

Reply:
> Ah, I thought the plan was to rename everything to `cosmossdk.io/x/{nft,staking,etc}` eventually. It's breaking, but it's also an easy search&replace.

Ah I didn't know!

Reply:
We had a quick chat during the SDK call, unless there's strong opinion against, we're going towards using the vanity url everywhere.

Reply:
I think for things like staking, we can cut a version with the old path then the new versions get the new path and people have to migrate. NFT is less hectic so breaking it is fine imo.